src/js/components/Image/__tests__/Image-test.js

```diff
@@ -28,3 +28,13 @@ test('Image fit renders', () => {
   const tree = component.toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+test('Image opacity renders', () => {
+  const component = renderer.create(
+    <Grommet>
+      <Image opacity="weak" src={SRC} />
```

Comment:
can we add tests for all the cases? like weak medium strong true false and a custom value?